packages/openbox/build.sh

```diff
@@ -36,6 +36,8 @@ etc/xdg/openbox/rc.xml
 "
 
 termux_step_post_make_install() {
+	sed -i "s|@TERMUX_PREFIX@|$TERMUX_PREFIX|g" ${TERMUX_PKG_BUILDER_DIR}/scripts/openbox-{autostart,session,xdg-autostart}
```

Comment:
Do not in-place edit the files in `$TERMUX_PKG_BUILDER_DIR`.

Reply:
Then i will edit it in `$TERMUX_PREFIX`, where the file has been copied.

Reply:
So in what way i will do this?

In this way (Each in new line)
```sh
cp -f "${TERMUX_PKG_BUILDER_DIR}/scripts/openbox-session" "${TERMUX_PREFIX}/bin/openbox-session"
sed -i "s|@TERMUX_PREFIX@|$TERMUX_PREFIX|g" "${TERMUX_PREFIX}/bin/openbox-session"
chmod 755 "${TERMUX_PREFIX}/bin/openbox-session"
```

Or this in the end:
```sh
sed -i "s|@TERMUX_PREFIX@|$TERMUX_PREFIX|g" ${TERMUX_PREFIX}/bin/openbox-{autostart,session,xdg-autostart}
```